src/main/java/slimeknights/tconstruct/shared/block/SlimeBlock.java

```diff
@@ -61,6 +69,44 @@ public int getBallColor() {
       return this.ballColor;
     }
 
+    public Tag<Item> getSlimeBallTag() {
+      return slimeBallTag;
+    }
+
+    public Food getSlimeFood(SlimeType type) {
```

Comment:
Toss some JavaDoc headers on these methods. I like to have all methods either have a `@Override` or a javadoc. I know not all of the project follows that, but if we add java docs for new methods it will be less work when I go back and add javadocs for all that we missed.